### PR TITLE
core(stack-packs): add Drupal pack

### PR DESCRIPTION
**Pull request by moerazem**
**Summary**
This PR brings Drupal stack pack to Lighthouse.

Note that the detection for Drupal is not yet released in `js-library-detector`. Therefore, this stack pack was tested based on the updated Drupal detection in [this PR](https://github.com/johnmichel/Library-Detector-for-Chrome/pull/163).

Here's a report with the JSON output of Lighthouse audit for www.tesla.com (a Drupal website). https://googlechrome.github.io/lighthouse/viewer/?gist=3dc26da0892873d860345d0411daa0b3

**Related Issues/PRs**
- Updated Drupal detection: https://github.com/johnmichel/Library-Detector-for-Chrome/pull/163

### CODE REVIEW

**Comment by moerazem** [comment]
@googlebot I signed it!

**Comment by housseindjirdeh** [comment]
Thanks for this @moerazem!

@wardpeet has been working on [improving](https://github.com/GoogleChrome/lighthouse-stack-packs/pull/38) how stack packs get integrated into Lighthouse directly so that we don't need to include them manually, but this still helps :)

**Comment by connorjclark** [comment]
With #10138 we are able to consume everything from the stack-packs module directly. If that lands first (probably won't) I don't think any of these changes will be needed–except the js lib detector update.

That said, we could probably go ahead and land this first (but CI is failing, looks like you need to run `yarn` to update the lockfile). Would be good to land this ASAP next week so we can get the strings translated.

**Comment by moerazem** [comment]
@connorjclark I have already added this stack pack to [GoogleChrome/lighthouse-stack-packs](https://github.com/GoogleChrome/lighthouse-stack-packs) in [#43](https://github.com/GoogleChrome/lighthouse-stack-packs/pull/43), which was merged 🎉.

I also opened this PR as I thought #10138 would take some time to land. So it would be awesome if this could make it for the next release.

Now the CI is failing for the number of duplicate strings checked by `yarn update:sample-json`. The string in `drupal.js` stack pack is in `UIStrings.efficient_animated_content`.

I could reword it, but I maybe it's better to get the same translations for this string.

**Comment by connorjclark** [comment]
@moerazem we just landed #10138, so now we are in a position to accept this PR... would you like to merge master back into your branch? Some files have been deleted fyi.

You'll probably just need to keep the js-library-detector upgrade and the stuff in `lighthouse-core/lib/stack-packs.js`

**Comment by moerazem** [comment]
@connorjclark I updated it. There's still a `stack-packs/packs/drupal.js` file. I have left it as I still see the other packs files in the master branch, even though I'm guessing this will be coming from the lighthouse-stack-packs npm package.

Let me know if this is no longer needed and I'll remove it.

File: stack-packs/packs/drupal.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+/**
```
Comment (connorjclark) [inline stack-packs/packs/drupal.js:1]: can delete

Comment (connorjclark) [reply stack-packs/packs/drupal.js:1]: ah, right those files were meant to be deleted, forgot to earlier: #11396

Comment (moerazem) [reply stack-packs/packs/drupal.js]: Sure. Deleted.

**Review by connorjclark — APPROVED** [review]

**Comment by connorjclark** [comment]
Thanks @moerazem!

**Pull request merged**